tools/automation/vendoring/__init__.py

```diff
@@ -29,10 +28,10 @@
 }
 
 # from time to time, remove the no longer needed ones
+_github_license = "https://github.com/{}/raw/master/LICENSE"
```

Comment:
I would leave this as it was. All it takes is one dependency with a `LICENSE.txt` to ruin it. Having to break the strings is annoying but it's the price we pay for keeping this kind of data in files with enforced line lengths.